src/charging-station/ocpp/1.6/OCPP16ResponseService.ts

```diff
@@ -693,15 +701,19 @@ export class OCPP16ResponseService extends OCPPResponseService {
         }
       }
     }
+
     if (
       connectorStatus?.status !== OCPP16ChargePointStatus.Available &&
-      connectorStatus?.status !== OCPP16ChargePointStatus.Preparing
+      connectorStatus?.status !== OCPP16ChargePointStatus.Preparing &&
+      connectorStatus?.status !== OCPP16ChargePointStatus.Reserved
```

Comment:
I think the issue is more related to the fact that sending a `StartTransaction` request after at local badge scan simulation is not putting the connector status in preparing state. And if the `StartTransaction` is refused server-side, the previous connector status is not properly restored.  

Would you redo that PR to implement it?

Reply:
Hi Jerome, found some time to test it with the UI. 

Regarding your first point:
- sending a StartTransaction request after at local badge scan simulation is not putting the connector status in preparing state
- Behaviour: Absolutely correct. If the connector is reserved, then the code breaks here.

Regarding your second point:
- if the StartTransaction is refused server-side, the previous connector status is not properly restored if a valid reservation is found
- Behaviour: Exactly, if the `StartTransaction` fails, the connector goes back to the `Available` state.

Reply:
> * sending a StartTransaction request after at local badge scan simulation is not putting the connector status in preparing state
> * Behaviour: Absolutely correct. If the connector is reserved, then the code breaks here.

The OCPP 1.6 specs are not clear about the connector preparing state sent at issuing a `StartTransaction`. But given the fact that: 
- `Reserved` to `Charging` is not a documented connector status transition (and the simulator code checking status transition will warn), 
- Putting a connector into `Preparing` status at `StartTransaction` sounds reasonable even if it's not the strict definition of preparing state in the specs.
the correct fix is to send the `Preparing` status at `StartTransaction` with code review for duplication sending.